ICA/auto_mycosnp_ICA.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import json
+import re
+import time
+import pandas as pd
+import numpy as np
+from datetime import date
+
+# This funciton uses the ica CLI tool to return info from ICA
+def icav_out(bashCommand):
+    process = subprocess.Popen(bashCommand.split(), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    tmp=output.decode("utf-8")
+    tmp=tmp.split('\n')
+    tmp.pop(0)
+    tmp.pop(-1)
+    tmp.pop(-1)
+    tmp=[x.replace('\t', ' ') for x in tmp]
+    return tmp
+
+# While loop is infinate will keep script running as long as it dosent error out.
+go=1
+while go == 1:
+# Sequencing runs are directly upload to ICA after basecalling into this project
+    bashCommand="icav2 projects enter BSSH UPHL_Bioinfo"
+    process = subprocess.Popen(bashCommand.split(" ",3), stdout=subprocess.PIPE)
+    process.communicate()
+#Folders of ICA Runs
+    bashCommand = "icav2 projectdata list --data-type FOLDER --file-name app.* --match-mode FUZZY"
+    tmp= icav_out(bashCommand)
+# ICA runs
+    ica_r=[]
+    for i in tmp:
+        ica_r.append(i.split(" ")[0])
+
+    runs_on_ica=[]
+    for i in range(len(tmp)):
+        runs_on_ica.append(tmp[i].split()[0].split('/')[2])
+# The plan is to download only the files that are commonly needed by Tom and use these directories as a way to way to moniter if anaylsis has completed.
+    runs_completed_analysis=os.listdir("/Volumes/IDGenomics_NAS/ICA/Runs_On_ICA")
+#This maybe unneeded but during testing I think the os package generates this file that needs to be removed from list
+    try:
+        runs_completed_analysis.remove('.DS_Store')
```

Comment:
It's best to keep the .DS_Store and @eaDir in case someone on their mac is looking in this directory